Vizzuality/landgriffon
# (chore) suggestions for deprecated GH actions stuff, Feb 2023 edition

### General description

Since I copypasted some of this for Foodscapes' GH actions recently and updated the bits that GH marked as deprecated, I thought I'd contribute my ~changes~ bugs back.

I've left `aws-actions/configure-aws-credentials@v1` alone even though they now support Node v16 via `@v1-node16`, because I think that at this stage and with a few months to go before GH fully removes the ability to use Node v12 for actions, it may be best to wait for a new major release from the `aws-actions` team, rather than only using the `@v1-node16` flavour "just" to squash deprecation warnings.

### Designs

N/A

### Testing instructions

All changed actions should run correctly as before.

Although I've tried to be careful and "just" backport the changes I did for the Foodscapes actions after copypasting the relevant LG actions, please cross check. If you come after me with a roaring chainsaw, I'll cast a disappearance spell.

## Checklist before merging
